### ISSUE TITLE
image not save

### ISSUE THREAD

**Issue by bigdevi** [chinalwb/Android-Rich-text-Editor#47]
try to insert image, in edit mode, image is show. but when it save. no image url in html.

**Comment by chinalwb** [comment]
Sorry for the issue. I'll check when I get time, this may need some time please aware..

**Comment by chinalwb** [comment]
@bigdevi Sorry for my late response. I could not reproduce, can you let me know your steps to reproduce? Thanks.

**Comment by tigerTuji** [comment]
Hi @chinalwb,
Image div is getting save properly. When try to reload it I am getting an exception:

java.lang.SecurityException: Permission Denial: opening provider com.android.externalstorage.ExternalStorageProvider from ProcessRecord{126da800 6308:com.app.learn/u0a61} (pid=6308, uid=10061) requires android.permission.MANAGE_DOCUMENTS or android.permission.MANAGE_DOCUMENTS.

Editor is working fine. Then I am saving html in database and on loading html from database is causing problem for image. 

Please look into it ASAP.

Thanks.

**Comment by chinalwb** [comment]
I'll check but looks the exception seems like you should add the specified permission to your manifest file.

**Comment by tigerTuji** [comment]
We cannot add permission for MANAGE_DOCUMENTS in apps which are not system app. It seems we cannot access gallery document by the URL that we have saved. Or requires to save image in local directory of app. 
Hope you will get a best solution soon.

**Comment by tigerTuji** [comment]
Are you not getting this issue while reopening ARE after saving some image? Your demo app also not included any special permission regarding this and even after adding MANAGE_DOCUMENTS permission, I am getting same error.

**Comment by chinalwb** [comment]
Yes, I know this. No schedule for fixing this in this community version. Waiting for the commercial version please.
Thanks!
